ert_storage/database_schema.py

```diff
@@ -24,6 +24,20 @@ class RecordClass(Enum):
     other = 3
 
 
+class PriorFunction(Enum):
+    normal = 1
+    triangular = 2
+    lognormal = 3
+    truncated_normal = 4
```

Comment:
`log_normal` or `truncatednormal` maybe? Would prefer having them on the same format at least 👍

Reply:
`truncnormal` and `lognormal` ;)